yaksh/urls.py

```diff
@@ -110,4 +110,6 @@
         views.download_assignment_file, name="download_quiz_assignment"),
     url(r'^manage/courses/download_yaml_template/',
         views.download_yaml_template, name="download_yaml_template"),
+    url(r'^manage/download_questionpaper/(?P<questionpaper_id>\d+)/$',
```

Comment:
please change the name from download_questionpaper to view_questionpaper or preview_questionpaper. Also, do the same for the html file download_questionpaper.html.